index.js

```diff
@@ -19,6 +19,161 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    const pastaAlfredo = {
+      title: "Pasta Alfredo",
+      level: "Easy Peasy",
+      ingredients: ["Pasta", "Cream", "Salt", "Pepper", "Cheese", "Flour"],
+      cuisine: "Italian",
+      dishType: "main_course",
+      image: "",
```

Comment:
Since you are not assigning `image` and `created` keys a specific value, you don't need to reference them when creating a new object from your model. And in this particular case, your `image` has a default value, once you pass that `empty string` value to it, you wont keep the default value from the model in your database.